server/src-lib/Hasura/GraphQL/Execute/Mutation.hs

```diff
@@ -161,51 +170,25 @@ convertMutationSelectionSet env logger gqlContext sqlGenCtx userInfo manager req
     >>= (mutationParser >>> (`onLeft` reportParseErrors))
 
   -- Transform the RQL AST into a prepared SQL query
-  txs <- for unpreparedQueries $ convertMutationRootField env logger userInfo manager reqHeaders (stringifyNum sqlGenCtx)
-  let txList = OMap.toList txs
-  case (mapMaybe takeTx txList, mapMaybe takeRemote txList) of
-    (dbPlans, []) -> do
-      let allHeaders = concatMap (snd . snd) dbPlans
-          combinedTx = toSingleTx $ map (G.unName *** fst) dbPlans
-      pure $ ExecStepDB (combinedTx, allHeaders)
-    ([], remotes@(firstRemote:_)) -> do
-      let (remoteOperation, varValsM') =
-            buildTypedOperation
-            G.OperationTypeMutation
-            varDefs
-            (map (G.SelectionField . snd . snd) remotes)
-            varValsM
-      if all (\remote' -> fst (snd firstRemote) == fst (snd remote')) remotes
-        then return $ ExecStepRemote (fst (snd firstRemote), remoteOperation, varValsM')
-        else throw400 NotSupported "Mixed remote schemas are not supported"
-    _ -> throw400 NotSupported "Heterogeneous execution of database and remote schemas not supported"
-  -- Build and return an executable action from the generated SQL
+  let userSession = _uiSession userInfo
+      rjCtx = (manager, reqHeaders, userInfo)
```

Comment:
I think `rjCtx` is pushing the limit of our [style guide’s section on naming](https://github.com/hasura/graphql-engine/blob/4062cdcbeabd10541f9e51e8b76b97f079f19b00/server/STYLE.md#naming). Can we expand it to `remoteJoinCtx`?

Reply:
This isn't new code (it just got moved around), and `rjCtx` is not a new name. But I do agree that it deserves a better name, so I'll do a search/replace for this as part of this PR.